atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -82,6 +85,31 @@ def __init__(self, tasker, workflow, koji_hub=None, koji_ssl_certs_dir=None,
         self._source_registry = get_source_registry(workflow, {})
         self._deep_manifest_list_inspection = get_deep_manifest_list_inspection(self.workflow,
                                                                                 fallback=True)
+        self.metadata = get_build_json().get("metadata", {})
+        self.osbs = None
+
+    def ignore_isolated_autorebuilds(self):
+        if not self.workflow.source.config.autorebuild.get('ignore_isolated_builds', False):
+            self.log.debug("ignoring_isolated_builds isn't configured, won't skip autorebuild")
+            return
+
+        base_koji_build = self.wait_for_parent_image_build(self._base_image_nvr)
+
+        is_isolated = base_koji_build['extra']['image'].get('isolated', False)
```

Comment:
Should we just forward `is_isolated` as `is_parent_isolated` to the delegate plugin somehow and let it handle the build cancellation now that the delegate plugin will have to run after this plugin anyway? Otherwise we'd be going beyond the original responsibilities of this plugin introducing openshift communication.

Reply:
yeah that is good idea, will do that

Reply:
done